plugins/modules/ome_configuration_compliance_baseline.py

```diff
@@ -745,23 +794,62 @@ def create_remediate_payload(noncomplaint_devices, baseline_info, rest_obj):
             "RunLater": False
         }
     }
+    if module.params.get("run_later"):
+        if validate_cron(module.params.get("cron")) is False:
+            module.exit_json(msg=INVALID_CRON_TIME_MSG, failed=True)
+        payload = {
+            "Id": baseline_info["Id"],
+            "Schedule": {
+                "RunNow": False,
+                "RunLater": True,
+                "Cron": module.params.get("cron")
+            }
+        }
+    elif module.params.get("staged_at_reboot"):
+        payload = {
+            "Id": baseline_info["Id"],
+            "IsStaged": True,
+            "Schedule": {
+            }
+        }
     if LooseVersion(ome_version) >= "3.5":
         payload["DeviceIds"] = noncomplaint_devices
     else:
         payload["TargetIds"] = noncomplaint_devices
     return payload
 
 
+def validate_cron(cron_string):
+    cron_pattern = CRON_REGEX
+    if re.match(cron_pattern, cron_string) is not None:
```

Comment:
This snippet can be enhanced.

`return bool(re.match(cron_pattern, cron_string))`

Reply:
Done